content/launch-keyboard.md

```diff
@@ -0,0 +1,68 @@
+).
+
+## Firmware Updates
+
+Firmware updates for the Launch are handled using [fwupd](https://fwupd.org/) and our <u>Firmware Manager</u> can be used to apply the update. When you get a notification about a firmware update you will need to unlock the device using Fn+Esc (note that this will be the default keys even if you remapped them in the Keyboard Configurator) before you can apply the update. When the keyboard is unlocked you will see the following keys light up:
+
+- U N L O C K E D
+
+Once the keyboard is in this mode you can apply the firmware update, when the update is complete you will need to unplug and re-plug it in to exit this mode.
+
+## Keyboard Configurator
+
+You can download the `AppImage` for Ubuntu and other OSes and use the <u>Pop!\_Shop</u> for Pop!\_OS. Use this [link](https://github.com/pop-os/keyboard-configurator/releases
+) to download the application.
+
+You can also download it from the Arch Linux AUR. The main version is built from latest releases which is recommended for most users which you can find [here](https://aur.archlinux.org/packages/keyboard-configurator/
+) and the git version is built from the latest commits to the project (this version will have the very latest fixes and features) which can be found [here](https://aur.archlinux.org/packages/keyboard-configurator-git/
+).
+
+## Remapping Keys
+
+<video width="900" height="900" controls>
+  <source src="/files/launch-keyboard/remapping-function-keys.webm" type="video/mp4">
+</video>
+
+The keys are defined in the firmware of your keyboard and you can remap the keys using the <u>Keyboard Configurator</u>. After opening the <u>Keyboard Configurator</u>, you can then select the key you would like to configure, and select an item from the available items below the keyboard showing in the application. I am attaching a video showing the remapping process. Please note the "Layer 1" and "Layer 2" at the top of the configurator. Layer 1 is what the keys will do normally, then Layer 2 shows what the keys do when "Fn" is held down.
+
+## Changing LEDs
+
+<video width="900" height="900" controls>
+  <source src="/files/launch-keyboard/video-soothing.mp4" type="video/mp4">
+</video>
+
+In the <u>Keyboard Configurator</u> you can change the LEDs in the 'LEDs' section of the application by changing the 'Layer Color Pattern' using the dropdown.
+
+## Tech Docs
+
+We have documentation that goes over changing the keycaps, switches and more in the tech docs which can be found [here](https://tech-docs.system76.com/models/launch_1/repairs.html).
```

Comment:
> We have documentation that goes over changing the keycaps, switches and more in the tech docs which can be found [here]

A comma is required before "which." I would also suggest an optional Oxford comma after "switches."